### PR TITLE
Refactor/pool card

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206703663957530

### CODE REVIEW

File: views/pools/pool-card/index.tsx [hunk 1]
```diff
@@ -15,38 +18,43 @@ const PoolCard: FC<PoolCardProps> = ({ tokens, dex, poolObjectId }) => {
...
+        bg="lowestContainer"
+        borderRadius="xs"
+        flexDirection="column"
+        justifyContent="center"
+        width={['100%', '100%', '100%', '100%', '21rem']}
+        border="0.063rem solid"
+        borderColor="outlineVariant"
+        transition="all 300ms ease-in-out"
+        nHover={{
+          borderColor: '#76767A',
```
Comment (KipandaJr) [inline views/pools/pool-card/index.tsx:35]: Check this color on `ui-kit`

File: views/pools/pool-card/pool-card-header.tsx [hunk 2]
```diff
@@ -11,44 +10,68 @@ const PoolCardHeader: FC<PoolCardHeaderProps> = ({
...
+        <a key={v4()} target="_blank" href={url} rel="noreferrer">
+          <Tag
+            px="0"
+            py="0"
+            mb="2xs"
+            mr="2xs"
+            size="small"
+            height="1.4rem"
+            variant="outline"
           >
```
Comment (KipandaJr) [inline views/pools/pool-card/pool-card-header.tsx:54]: this tags r not a link on info value like `Volatile` the link has only on Dex name

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image
 
the page break on this point `1292px`

File: views/pools/pool-card/index.tsx (hunk 1 above)
Comment (atilasousa) [reply views/pools/pool-card/index.tsx:35]: dosen't exist in colors

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image
not solved the responsiviness

File: views/pools/pool-card/pool-card-header.tsx [hunk 3]
```diff
@@ -11,44 +10,67 @@ const PoolCardHeader: FC<PoolCardHeaderProps> = ({
...
-        <ArrowObliqueSVG maxHeight="1.5rem" maxWidth="1.5rem" width="100%" />
-      </Button>
-    </Link>
+    <Box display="flex" flexWrap="wrap" alignItems="center">
+      <a target="_blank" href={dexUrl} rel="noreferrer">
+        <Tag
+          p="0"
+          mr="2xs"
+          mb="2xs"
+          gap="0"
```
Comment (MarioBatalha) [inline views/pools/pool-card/pool-card-header.tsx:22]: This gap and padding are unnecessary

File: views/pools/pool-card/pool-card-header.tsx [hunk 4]
```diff
@@ -11,44 +10,67 @@ const PoolCardHeader: FC<PoolCardHeaderProps> = ({
...
+        >
+          <Typography size="small" variant="label">
+            {name}
+          </Typography>
+        </Tag>
+      </a>
+      {tags?.map(({ name }) => (
+        <Tag
+          key={v4()}
+          px="0"
```
Comment (MarioBatalha) [inline views/pools/pool-card/pool-card-header.tsx:47]: Are u sure that this px and py is necessary?

File: views/pools/pool-card/pool-card-header.tsx [hunk 5]
```diff
@@ -11,44 +10,67 @@ const PoolCardHeader: FC<PoolCardHeaderProps> = ({
...
+          <Typography size="small" variant="label">
+            {name}
+          </Typography>
+        </Tag>
+      ))}
+    </Box>
+    <Button
+      mb="auto"
+      isIcon
+      opacity="0"
```
Comment (MarioBatalha) [inline views/pools/pool-card/pool-card-header.tsx:64]: You could add here some transition effect

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by KipandaJr — APPROVED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**